index.html

```diff
@@ -21,19 +21,19 @@
         <tbody>
             <!--           first person -->
```

Comment:
Hi @Johnbelieve you are very close, however, because another PR was merged into the master branch while your's was waiting you would need to run git fetch to have the latest version of the file on your machine.

Also, you need to fill in your details once, not twice.

After doing this, you should review if there's any. conflict in the file, fix them and resubmit your PR.